From d792aebfa2daeb58ac6a669ce328ad05e9e4112c Mon Sep 17 00:00:00 2001
From: Jorgen Jonsson <jorgen.jonsson@stericsson.com>
Date: Mon, 5 Nov 2012 16:09:31 +0100
Subject: [PATCH 470/951] ARM: ux500: fixup magnetometer pins

GPIO31 and GPIO32 (CTS/RTS) are handled by the magnetometer and
shall not simultaneously be controlled by UART2.

Reported-by: Peter Nessrup <peter.nessrup@stericsson.com>
Signed-off-by: Jorgen Jonsson <jorgen.jonsson@stericsson.com>
Reviewed-by: Patrice Chotard <patrice.chotard@stericsson.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
---
 arch/arm/mach-ux500/board-mop500-pins.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/mach-ux500/board-mop500-pins.c b/arch/arm/mach-ux500/board-mop500-pins.c
index f9ef2f4..5770ed0 100644
--- a/arch/arm/mach-ux500/board-mop500-pins.c
+++ b/arch/arm/mach-ux500/board-mop500-pins.c
@@ -148,15 +148,10 @@ static struct pinctrl_map __initdata mop500_family_pinmap[] = {
 	 * The setting on GPIO31 conflicts with magnetometer use on hrefv60
 	 */
 	/* default state for UART2 */
-	DB8500_MUX("u2ctsrts_c_1", "u2", "uart2"),
-	DB8500_PIN("GPIO31_V3", in_pu, "uart2"), /* CTS */
-	DB8500_PIN("GPIO32_V2", out_hi, "uart2"), /* RTS */
 	DB8500_MUX("u2rxtx_c_1", "u2", "uart2"),
 	DB8500_PIN("GPIO29_W2", in_pu, "uart2"), /* RXD */
 	DB8500_PIN("GPIO30_W3", out_hi, "uart2"), /* TXD */
 	/* Sleep state for UART2 */
-	DB8500_PIN_SLEEP("GPIO31_V3", in_wkup_pdis, "uart2"),
-	DB8500_PIN_SLEEP("GPIO32_V2", out_hi_wkup_pdis, "uart2"),
 	DB8500_PIN_SLEEP("GPIO29_W2", in_wkup_pdis, "uart2"),
 	DB8500_PIN_SLEEP("GPIO30_W3", out_wkup_pdis, "uart2"),
 	/*
-- 
1.8.0

